src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -100,8 +100,7 @@ class NativeSessionStorage implements SessionStorageInterface
     public function __construct(array $options = array(), $handler = null, MetadataBag $metaBag = null)
     {
         $options += array(
-            // disable by default because it's managed by HeaderBag (if used)
-            'cache_limiter' => '',
+            'cache_limiter' => 'private_no_expire',
```

Comment:
This was implicit before because caches usually don't store responses with "Set-Cookie"; but now that we don't send the cookie, we must tell proxies to not store the page, otherwise eg. flashes will never be shown (as the cached response will be served).
Of course, apps can take over and set another Cache-Control, since HeaderBag applies after the session started.